docker-compose.yml

```diff
@@ -1,9 +1,46 @@
 services:
   zed:
     build: .
+    container_name: zed_web
+    command: bash ./bashes/RunBash.sh
+    restart: always
     volumes:
-      - db:/app/db.sqlite3
+      - .:/code
```

Comment:
By mounting the code as a volume, we can make changes to the code on our host machine and see those changes reflected immediately in the running container, without the need to rebuild the image. This significantly accelerates development cycles and debugging processes.

Reply:
You should write a Docker Compose file for the production environment. Developers can make any changes for their ease of development in their own environment in the project.

Reply:
You're right about separate environments. Currently, ./code includes both application code and likely some development-specific things. While I want developers to have a flexible development environment, creating a separate docker-compose.prod.yml file specifically for production makes sense. This would exclude development configurations and potentially have stricter settings.

Reply:
I think the owner should consider a good practice for this issue 
@EngRobot33